### PR TITLE
Updating readme and webhook tempate to capture changes made for volumesnapshotclasses

### PR DESCRIPTION
**Pull request by shawn-hurley**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Updating the readme based on the new kubeconfig flag for the webhook and webhook template such that the webhook will be used for volumesnapshotclasses as well

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ Read more about how to install the example webhook [here](deploy/kubernetes/webh
 
 * `--port`: Secure port that the webhook listens on (default 443)
 
+* `--kubeconfig <path>`: Path to Kubernetes client configuration that the webhook uses to connect to Kubernetes API server. When omitted, default token provided by Kubernetes will be used. This option is useful only when the snapshot controller does not run as a Kubernetes pod, e.g. for debugging.
```
Comment (jsafrane) [inline README.md:112]: Please add also a sentence or two to "Validating Webhook" chapter above that snapshot classes are validated too and how it work (only one class can be default for each CSI driver).

Comment (shawn-hurley) [reply README.md:112]: Update, let me know if the new section makes sense?

Comment (jsafrane) [reply README.md:112]: makes sense, thanks!

**Review by jsafrane — COMMENTED** [review]
/lgtm

File: deploy/kubernetes/webhook-example/admission-configuration-template [hunk 2]
```diff
@@ -8,7 +8,7 @@ webhooks:
   - apiGroups:   ["snapshot.storage.k8s.io"]
     apiVersions: ["v1", "v1beta1"]
     operations:  ["CREATE", "UPDATE"]
-    resources:   ["volumesnapshots", "volumesnapshotcontents"]
+    resources:   ["volumesnapshots", "volumesnapshotcontents", "volumesnapshotclasses"]
```
Comment (xing-yang) [inline deploy/kubernetes/webhook-example/admission-configuration-template:11]: Please add this to the release note. Any time we add a new RBAC rule, it requires a major version bump.

**Comment by xing-yang** [comment]
Can you backport this to release-6.0?

File: deploy/kubernetes/webhook-example/admission-configuration-template (hunk 2 above)
Comment (jsafrane) [reply deploy/kubernetes/webhook-example/admission-configuration-template:11]: This is not RBAC rule. But I added some text about it to the first commit.

**Comment by shawn-hurley** [comment]
Backport PR: https://github.com/kubernetes-csi/external-snapshotter/pull/700

**Pull request merged**